edge-fqm/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>folio-integration-tests</artifactId>
+        <groupId>org.folio</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>edge-fqm</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.folio</groupId>
+            <artifactId>common</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>org.folio</groupId>
+            <artifactId>testrail-integration</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+    </dependencies>
```

Comment:
Would anyone be upset if these dependencies got moved to the `folio-integration-tests` parent POM's `dependendencyManagement` section? That way, we could remove all of the versions from the children POMs and manage the versions across the board in 1 place